sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/inprocess/ExecutorServiceParallelExecutor.java

```diff
@@ -126,7 +126,7 @@ public void scheduleConsumption(
       @Nullable final CommittedBundle<T> bundle,
```

Comment:
The passed transform is a root transform and will consume no input (but may produce output)